pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -241,6 +241,7 @@ rec {
                    && (if config.checkMetaRecursively or false
                        then lib.all (d: d.meta.available or true) references
                        else true);
+          supported = validity.supported or false;
```

Comment:
I think this does the wrong thing in case of unfree license.

Reply:
Good point, should allowUnfree influence the enable condition or not?